src/api.js

```diff
@@ -0,0 +1,76 @@
+const form = document.getElementById('form');
+let gameId = '';
+
+async function submitForm(event) {
+  event.preventDefault();
+
+  const nameInput = document.getElementById('name');
+  const scoreInput = document.getElementById('score');
+
+  const data = {
+    name: nameInput.value,
+    score: scoreInput.value,
+    user: 'John Doe',
+  };
+
+  try {
+    const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        name: 'Soccer LeadorBoard',
+      }),
+    });
+
+    const responseData = await response.json();
+    const id = responseData.result.match(/ID: (\w+)/)[1];
+    gameId = id;
+
+    const scoresResponse = await fetch(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${id}/scores`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+
+    const result = await scoresResponse.json();
+    console.log('Score posted successfully:', result);
+  } catch (error) {
+    console.error('Error:', error);
+    alert('Something went wrong');
+  }
+}
```

Comment:
- Good job done so far, but there is an issue with your code. You seem to be generating a new ID each time a new score is submitted. Therefore I get only one score each time I refresh the scoreboard. Please fix this 🙏 
![esrhserh](https://github.com/paulshaun-kdtk/leaderboard/assets/73607512/d48eda9e-1f9b-4605-bb11-105833983859)